src/Uno.UI/UI/Xaml/Controls/CalendarView/CalendarViewGeneratorHost.cs

```diff
@@ -350,7 +350,7 @@ internal DateTime GetDateAt(uint index)
 		internal int CalculateOffsetFromMinDate(DateTime date)
 		{
 			var pIndex = 0;
-			DateTime estimatedDate = m_lastVisitedDateAndIndex.first.ToUniversalTime();
+			DateTime estimatedDate = m_lastVisitedDateAndIndex.first; //.ToUniversalTime(); - UNO
```

Comment:
@dr1rrb is there an abstraction in the runtime tests for this control to help timezone changes?

Reply:
Don't think so

Reply:
Looking at the code, this is probably the way to go : https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.calendarview.calendaridentifier?view=winrt-22621

Reply:
@dr1rrb Could we merge without a test?

Or, could we do [this](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/change-time-zone?view=azure-devops&tabs=preview-page)?